src/commands/registerCommands.ts

```diff
@@ -40,6 +40,7 @@ export function registerCommands(): void {
     registerCommand('ms-azuretools.loadMore', async (context: IActionContext, node: AzExtTreeItem) => await ext.helpTree.loadMore(node, context));
     registerCommand('ms-azuretools.reportIssue', reportIssue);
     registerCommand('ms-azuretools.reviewIssues', reviewIssues);
+    registerCommand('ms-azuretools.openWalkthrough', () => commands.executeCommand('workbench.action.openWalkthrough', `ms-azuretools.vscode-azureresourcegroups#azure-get-started`));
```

Comment:
The Docker extension actually includes this command in the package.json and suppresses it from showing in the command palette. Is that needed to make this work or does it work fine without that?

Reply:
It's actually not needed. It doesn't show up in the command palette.